_maps/cyberiad.dm

```diff
@@ -24,7 +24,14 @@ z7 = empty
 
         #define MAP_FILE "cyberiad.dmm"
         #define MAP_NAME "NSS Cyberiad"
-        #define MAP_TRANSITION_CONFIG list(MAIN_STATION = CROSSLINKED, CENTCOMM = SELFLOOPING, TELECOMMS = CROSSLINKED, CONSTRUCTION = CROSSLINKED, MINING = CROSSLINKED, DERELICT = CROSSLINKED, EMPTY_AREA = CROSSLINKED)
+        #define MAP_TRANSITION_CONFIG list(\
```

Comment:
Isn't this a #define? If I remove this, then the #define won't include any of the other lines.